### ISSUE TITLE
Plans for Angular 2?

### ISSUE THREAD

**Issue by coryschires** [bvaughn/angular-form-for#182]
We've been using angular-form-for with Angular 1 for severa months. It's been very helpful, especially the ability to create custom, easily-reusable form elements (e.g. file uploaded, date pickets, etc).

As we begin upgrading to Angular 2, I'm wondering how angular-form-for will fit into the picture (if at all). So, I have some questions:

* Do you plan on updating this library to support  Angular 2?
* Do you anticipate that the upgrade will require significant changes to your public API?
* Assuming you will upgrade, any idea of rough timeline?

Overall, we're trying trying to determine the ideal way to write forms in Angular 2 and would like to know if this is a library we can continue to count on or if we'd be better off looking at other options.

Thanks for making this library! It's been great.

**Comment by bvaughn** [comment]
Hi @coryschires. Nice to hear from you again. :)

My work situation has changed pretty drastically over the last 6 months. I was working at Google with Angular but now I'm at a startup working with React. Over the last few months I haven't been following the Angular 1.x to 2.x transition very closely.

I did initially start working on an indirect port dubbed [forms-js](https://github.com/bvaughn/forms-js). The plan was to support a variety of frameworks (including Angular 2) with a shared base validation library. The validation library was pretty solid IMO but... I didn't make it much further than that before I got interrupted by a busy work schedule.

I still have an interest in form-generation and would kind of like to see the forms-js idea through but it's hard to stay on top of a framework when I'm not using it during my day job. If your company would be interested in discussing some sort of contractor arrangement, we should chat. But if it's just a matter of me finding free time, I'm afraid some of my other React libraries are more the focus of my attention lately.

Feel free to reach out to me via email if you'd like to talk more. :)
